testing.txt

```diff
@@ -0,0 +1 @@
+'' 
```

Comment:
I think this shouldn't be empty right? There should be something between the quotes.